hack/vendor.sh

```diff
@@ -59,6 +59,8 @@ clone git github.com/go-fsnotify/fsnotify v1.2.0
 
 clone git github.com/go-check/check 64131543e7896d5bcc6bd5a76287eb75ea96c673
 
+clone git github.com/Azure/go-ansiterm
```

Comment:
It's a much smaller pull request now.  And in the mean time I've fixed all known bugs.  There are no panics or odd behavior in any known real world scenario.  The synthetic test vttest doesn't pass, but iTerm doesn't pass that test either.